packages/application-components/src/theming.ts

```diff
@@ -0,0 +1,25 @@
+import kebabCase from 'lodash/kebabCase';
+import { designTokens } from '@commercetools-uikit/design-system';
+
+export const themesOverrides = {
+  default: {
+    paddingVerticalForPage: designTokens.spacingM,
+    paddingHorizontalForPage: designTokens.spacingM,
+  },
+  test: {
+    paddingVerticalForPage: designTokens.spacingXl,
+    paddingHorizontalForPage: '40px',
+  },
+};
```

Comment:
Alternative approach, given that these tokens are mainly for the "migration" rather than the actual theme.

In the docs-kit (if you remember) I injected some inline css variables to the parent container, which can then be referenced in the styles.
https://github.com/commercetools/commercetools-docs-kit/blob/e873a23a6ea9647bb09d9e43d9c535880d616d48/packages/gatsby-theme-docs/src/layouts/internals/layout-page-with-hero.js#L89-L91

Maybe we can do the same here? So we're not really extending the theme but simply injecting custom css variables. The values can then be determined for example based on the feature flag (or something similar).